sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1514,8 +1521,10 @@ class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     def __init__(
         self,
         eps=1e-3,
-        n_alphas=100,
-        alphas=None,
+        ############ TODO:
+        ############ Need to change _parameter_constraints dictionary
```

Comment:
put a `TODO(1.9): remove deprecated options` where the dictionary exists instead